### PR TITLE
Airplane create af

### PR DESCRIPTION
**Pull request by AutumnFoto**
## Description
airplane create 

## Related Issue
#7 

## How Can This Be Tested?
git fetch airplaneCreateAF

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/javascripts/helpers/data/planeData.js [hunk 1]
```diff
@@ -19,6 +19,14 @@ const getPlanes = () => new Promise((resolve, reject) => {
     .catch((error) => reject(error));
 });
 
+const addPlane = (data) => axios
+  .post(`${baseUrl}/airports.json`, data)
```
Comment (chris-calhoun) [inline src/javascripts/helpers/data/planeData.js:23]: should be the planes.json file.

**Review by chris-calhoun — COMMENTED** [review]
Type is show as undefined on the DOM. Can you add a type field to the form?

**Review by chris-calhoun — APPROVED** [review]
👍

**Review by BaileyEDennis — APPROVED** [review]
👍

**Pull request merged**